corehq/apps/linked_domain/view_helpers.py

```diff
@@ -111,6 +113,24 @@ def get_upstream_and_downstream_keywords(domain):
     return upstream_list, downstream_list
 
 
+def get_upstream_and_downstream_ucr_expressions(domain):
+    """
+    Return 2 lists of ucr expressions
```

Comment:
I'm regretting my liberal use of the word "list" in these docstrings, but don't expect any changes here.